requirements.txt

```diff
@@ -124,7 +124,7 @@ requests-oauthlib==0.7.0
 requests-toolbelt==0.7.0
 requestsexceptions==1.1.3
 requirements-detector==0.5.2
-responses==0.5.1
+responses==0.10.4
```

Comment:
I updated the method that does a request to rabbitmq so it will [raise a RabbitMQAPIError](https://github.com/open-craft/opencraft/blob/josuebc/BB-692/instance/models/mixins/rabbitmq.py#L155) with a descriptive message because before there wasn't a message and it can become a bit hard to debug something like that. The old `responses` library did not mock the `reason` attribute in a `requests` Response instance so the test would fail with an `AttributeError`.
And `requests` is pinned to `2.11.1` which has a [Response model](https://github.com/requests/requests/blob/v2.11.1/requests/models.py#L545) with the `reason` attribute.